### ISSUE TITLE
Non matching serde version compatibility

### ISSUE THREAD

**Issue by pingiun** [RReverser/serde-xml-rs#78]
The README currently reads "xml-rs based deserializer for Serde (compatible with 0.9+)", while the project description says "xml-rs based deserializer for Serde (compatible with 1.0+)"

**Comment by alexschrod** [comment]
How has this not been dealt with in nearly two years?!

**Comment by punkstarman** [comment]
@alexschrod, I would be happy to review your pull request if you would be so kind as to lend a hand.

This open-source project, like so many others, is advanced only by the contributions of people in their spare time.

**Comment by alexschrod** [comment]
I agree with you under most circumstances, especially when it comes to implementing new features or fixing complicated bugs; things that would require a person to sit down and figure out the project and then contribute a relatively large and satisfying "chunk" of useful code in the end.

But in this case, as an outsider, I would have to spend a comparably much larger amount of time and effort to figure out whether this project is compatible with 0.9+ or 1.0+, whereas somebody who works on this project regularly and is familiar with its innards should be able to do the same with _much_ less effort.

This is not the kind of issue where most projects would be asking for outsider contributions. The investment is large and the reward is minimal.
